### PR TITLE
Add a point about running storybook for e2e tests.

### PR DESCRIPTION
**Pull request by grundleborg**
#### Summary
Add missing step from cypress guide about running the storybook.

### CODE REVIEW

**Review by saturninoabril — APPROVED** [review]
Thank you @grundleborg!

File: site/content/contribute/webapp/end-to-end-tests.md [hunk 1]
```diff
@@ -18,6 +18,7 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
 1.  Run a local server instance by initiating `make run` to the mattermost-server folder. Confirm that the instance has started successfully.
     - Run `make test-data` to preload your server instance with initial seed data.
     - Each test case should handle the required system or user settings, but in case you encounter an unexpected error while testing, you may want to run the server with default config based on `mattermost-server/config/default.json`.
+    - In another terminal, run `npm run storybook` in the `mattermost-webapp` directory. This is required for the Cypress tests which run against the Storybook widgets to pass.
```
Comment (justinegeffen) [inline site/content/contribute/webapp/end-to-end-tests.md:21]:
```suggestion
    - In another terminal, in the `mattermost-webapp` directory, run `npm run storybook`. This is required for the Cypress tests which run against the Storybook widgets to pass.
```

File: site/content/contribute/webapp/end-to-end-tests.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
 1.  Run a local server instance by initiating `make run` to the mattermost-server folder. Confirm that the instance has started successfully.
```
Comment (justinegeffen) [inline site/content/contribute/webapp/end-to-end-tests.md:18]:
```suggestion
1.  Run a local server instance by initiating `make run` to the `mattermost-server` directory. Confirm that the instance has started successfully.
```

**Review by justinegeffen — APPROVED** [review]

File: site/content/contribute/webapp/end-to-end-tests.md (hunk 2 above)
Comment (grundleborg) [inline site/content/contribute/webapp/end-to-end-tests.md:18]: If we're going to alter other parts of the instructions in this PR, I'd suggest:

```suggestion
1.  Launch a local Mattermost instance by running `make run` in the mattermost-server directory. Confirm that the Mattermost instance has started successfully.
```

Comment (grundleborg) [reply site/content/contribute/webapp/end-to-end-tests.md:18]: @justinegeffen ^

Comment (grundleborg) [reply site/content/contribute/webapp/end-to-end-tests.md:18]: @justinegeffen ping on this - would be good to get it merged.

**Review by justinegeffen — APPROVED** [review]

**Comment by justinegeffen** [comment]
Thanks, @grundleborg, apologies for the delay. It looks good to me. :)

File: site/content/contribute/webapp/end-to-end-tests.md [hunk 3]
```diff
@@ -15,9 +15,10 @@ Not familiar with Cypress? Here is some documentation that will help you get sta
 ## Quick Overview on How to Run E2E Testing
 
 ### On Your Local Development Machine
-1.  Run a local server instance by initiating `make run` to the mattermost-server folder. Confirm that the instance has started successfully.
+1.  Launch a local Mattermost instance by running `make run` in the mattermost-server directory. Confirm that the Mattermost instance has started successfully.
```
Comment (justinegeffen) [inline site/content/contribute/webapp/end-to-end-tests.md:18]:
```suggestion
1.  Launch a local Mattermost instance by running `make run` in the `mattermost-server` directory. Confirm that the Mattermost instance has started successfully.
```

**Comment by grundleborg** [comment]
Merged through https://github.com/mattermost/mattermost-developer-documentation/pull/687

**Pull request merged**